stablehlo/reference/Ops.cpp

```diff
@@ -282,9 +278,9 @@ Tensor evalSliceOp(const Tensor &operand, ArrayRef<int64_t> startIndices,
   Tensor result(resultType);
   for (auto resultIt = result.index_begin(); resultIt != result.index_end();
        ++resultIt) {
-    SmallVector<int64_t> operandIdx;
+    Index operandIdx(operand.getType().getRank());
     for (auto dim = 0; dim < operand.getType().getRank(); ++dim)
-      operandIdx.push_back(startIndices[dim] + (*resultIt)[dim] * strides[dim]);
+      operandIdx[dim] = startIndices[dim] + (*resultIt)[dim] * strides[dim];
```

Comment:
Can we rewrite this as something like `auto operandIdx = startIndices + *resultIt * strides`?